frontend/packages/ceph-storage-plugin/src/components/namespace-store/create-namespace-store.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { RouteComponentProps } from 'react-router';
+import { Title } from '@patternfly/react-core';
+import { history } from '@console/internal/components/utils/router';
+import { BreadCrumbs, resourcePathFromModel } from '@console/internal/components/utils';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { ListKind } from 'public/module/k8s/types';
+import CreateNamespaceStoreForm from './create-ns';
+import '../common/noobaa-store/create-bs-ns.scss';
+
+const CreateNamespaceStore: React.FC<CreateNamespaceStoreProps> = ({ match }) => {
+  const { t } = useTranslation();
+  const { ns, appName } = match.params;
+  const [csv, csvLoaded, csvErr] = useK8sGet<ListKind<ClusterServiceVersionKind>>(
+    ClusterServiceVersionModel,
+    '',
```

Comment:
```suggestion
    appName,
```

Reply:
I am keeping it the same. CSV obj is undefined when passing the appName resulting in error